src/geometry_controller/__init__.py

```diff
@@ -492,12 +492,16 @@ def get_list_weight(element_id: int) -> float:
     """
 
 def get_volume(element_id: int) -> float:
-    """Gets the element real volume 
+    """Gets the element raw volume 
```

Comment:
Can we use the term 'rough' instead of 'raw'? This is consistent with the terminology used elsewhere in the 3D software.

Reply:
No problem. Done for both.